tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/client/TemporaryDestinationTest.java

```diff
@@ -81,7 +82,7 @@ public void testTemporaryQueueLeak() throws Exception {
 
          tempQueue.delete();
 
-         assertFalse(conn.containsKnownDestination(SimpleString.toSimpleString(tempQueue.getQueueName())));
+         assertFalse(((ActiveMQDestination) tempQueue).isCreated());
```

Comment:
This test fails because `ActiveMQDestination.delete()` doesn't call `setCreated(false)`.